04-scan-cluster/solution/system-discovery-role.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  creationTimestamp: 2019-10-22T08:26:37Z
```

Comment:
I think we can cleanup the generated fields? 🤔

Reply:
Of course !